scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -168,6 +168,60 @@ def shortest_path(csgraph, method='auto',
     >>> predecessors
     array([-9999,     0,     0,     1], dtype=int32)
 
+    >>> graph = [
+    ... [0, 0, 7, 0],
+    ... [0, 0, 8, 5],
+    ... [7, 8, 0, 0],
+    ... [0, 5, 0, 0]
+    ... ]
+    >>> graph = csr_array(graph)
+    >>> print(graph)
+    <Compressed Sparse Row sparse array of dtype 'int64'
+    	with 6 stored elements and shape (4, 4)>
+    	Coords	Values
+    	(0, 2)	7
+    	(1, 2)	8
+    	(1, 3)	5
+    	(2, 0)	7
+    	(2, 1)	8
+    	(3, 1)	5
+
+    >>> sources = [0, 2]
+    >>> dist_matrix, predecessors = shortest_path(csgraph=graph, directed=False, indices=sources, return_predecessors=True)
+    >>> dist_matrix
+    array([[ 0., 15.,  7., 20.],
+           [ 7.,  8.,  0., 13.]])
+    >>> predecessors
```

Comment:
Up until this point the example is exactly the same as the old one, we could probably just remove the old one for conciseness?

Reply:
> we could probably just remove the old one for conciseness?

Makes sense. Would prefer removing. The new example is better IMO.